package.json

```diff
@@ -133,7 +133,7 @@
 		"@wordpress/dependency-extraction-webpack-plugin": "3.2.1",
 		"@wordpress/dom": "3.2.7",
 		"@wordpress/e2e-test-utils": "6.0.2",
-		"@wordpress/e2e-tests": "^3.1.1",
+		"@wordpress/e2e-tests": "3.1.1",
```

Comment:
Any reason this was also changed?

Reply:
Every other dependency is pinned to a specific version, I thought I'd do the same here as I noticed the package-lock.json was being updated every time I ran `npm i` locally.